app/controllers/cards_controller.rb

```diff
@@ -34,6 +34,16 @@ def update
     end
   end
 
+  def check
+    if @card.check_translation(check_card_params[:original_text])
+      flash[:notice] = "Правильно"
+      redirect_to root_path
+    else
+      flash.now[:error] = "Не правильно"
+      render 'home/random_card'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.